articles/azure-large-instances/workloads/epic/onboarding-requirements.md

```diff
@@ -0,0 +1,190 @@
+
+For setup instructions, see [Enable ExpressRoute Fast Path](#enable-expressroute-fast-path).
+
+To see the learned routes from ALI, one of the options is looking at the Effective Routes table of one of your VMs, as follows:
+
+1. In Azure portal, select any of your VMs (any connected to the Hub, or to a Spoke connected to the Hub which is connected to ALI for Epic), select **Networking**, select the network interface name, then select **Effective Routes**.
+
+2. Make sure to enable accelerated networking with all VMs connecting to ALI for Epic (link1) or (link2).
+
+3. Set up ALI for Epic solution based on your system requirements and take a system backup.  
+4. Take an OS backup.  
+5. Set up volume groups. (See [Create a volume group](create-a-volume-group.md).)  
+6. Set up a storage snapshot, backup, and data offload. (For detailed steps, see [Azure Large Instances NETAPP storage data protection with Azure CVO](ali-netapp-with-cvo.md)).
+
+> [!Note]
+> A storage snapshot should only be set up after all data-intensive work (for example, Endian conversions) are complete in order to avoid creating unnecessary snapshots while build work is in progress
+
+The Azure subscription you use for ALI deployments is already registered with the ALI resource provider by the Microsoft Operations team during the provisioning process.
+If you don't see your deployed Azure Large Instances under your subscription, register the resource provider with your subscription. For more information, see [Register the ALI resource provider](register-the-ali-resource-provider.md).
+
+### Enable ExpressRoute Fast Path
+
+Before you begin, install the latest version of the Azure resource manager power shell cmdlets, at least 4.0 or later.
+For more information about installing the power shell cmdlets, see [How to install Azure Powershell](https://learn.microsoft.com/powershell/azure/install-azure-powershell?view=azps-10.0.0).
+
+For more information, see these resources:
+
+* [Azure ExpressRoute overview](https://azure.microsoft.com/products/expressroute/)
+
+* [How to create a connection between your VPN Gateway and ExpressRoute circuit](https://learn.microsoft.com/shows/azure/expressroute-how-to-create-connection-between-your-vpn-gateway-expressroute-circuit) 
+
+
+* [How to set up Microsoft peering for your ExpressRoute circuit](https://learn.microsoft.com/shows/azure/expressroute-how-to-set-up-microsoft-peering-your-expressroute-circuit)
+
+### Authorizing  
+
+Ensure you have an authorization key for the express route (ER) circuit used for virtual gateway connection to ER circuit. 
+Also obtain ER circuit resource ID.
+
+If you don’t have this information, obtain the details from the circuit owner (these details are usually provided by the Microsoft team as part of provisioning request completion.
+Reach out to
+<a href=mailto:"AzureBMISupportEpic@microsoft.com">Microsoft support</a> in case of any inconsistencies).  
```

Comment:
@ryellajo12 , @imsunnymaghosh , we should remove the email address and replace it with the link to engage support given we'll have full CSS support by the time the doc is live, correct?

Reply:
Replace line 109 and 110 as : If you need Azure ALI Support, create a support ticket with Azure Customer Support.

Reply:
@imsunnymaghosh could you please provide the link you want used for this, thanks.

Reply:
Please link to ### Open a support request for ALI instances from articles/azure-large-instances/work-with-ali-in-the-azure-portal.md article. @suzizuber Cc: @imsunnymaghosh